layer/nrlf/consumer/fhir/r4/model.py

```diff
@@ -569,7 +569,7 @@ class DocumentReference(BaseModel):
         str,
         Field(
             description="The status of this document reference.",
-            pattern="[^\\s]+(\\s[^\\s]+)*",
+            pattern="^current$",
```

Comment:
The model shouldn't be updated like this as this file is auto-generated (only found this out recently while doing the Pydantic v2 ticket) . It should be updated in the API spec in `api/consumer/swagger.yaml`  and then regenerated with:

```
$ make generate-models
```

As this is the first model change we've done since moving to Pydantic v2, give us a shout if you want to talk over things.